### ISSUE TITLE
Make the style match furo theme

### ISSUE THREAD

**Issue by humitos** [readthedocs/sphinx-hoverxref#188]
When using sphinx-hoverxref with furo theme, it does not look great. It seems we need to add some adjustments via CSS as we do for some of the other themes. See

https://github.com/readthedocs/sphinx-hoverxref/blob/34c8a3c0d5bab718e1251d4449ab2a88892074bf/hoverxref/extension.py#L285-L287

![Screenshot_2022-05-31_15-36-48](https://user-images.githubusercontent.com/244656/171186822-2ab86e27-0731-4ccc-9cd4-ae22af6d246e.png)

(the screenshot is taken from this project https://crest.readthedocs.io/en/latest/user/configuration.html#configuration)

I'm cc'ing @pradyunsg just because he is the author of the furo theme 👋🏼

**Comment by pradyunsg** [comment]
I'm happy to add dedicated CSS for this in Furo, FWIW.

Furo already generates a CSS file for overriding extensions' styles, so this would work just fine with that and avoid an additional network request in those cases.

**Comment by humitos** [comment]
I opened #189 as a quick fix for the `h` html tags inside the tooltip which seems to work fine. However, as I don't have good CSS skills I don't know if it's the _correct_ way of doing it.

I'm 👍🏼 on adding CSS for this directly in Furo. Let me know if I can help somehow with that. I haven't checked Furo's code, but if you want me to open a PR there and continue the conversation on the PR itself, just point me to the file I need to start reading to make the changes 😄 

Do you think it is worth merging this PR and adding this small CSS code to the next release or it's better/cleaner to wait for the code to be part of Furo itself?

**Comment by pradyunsg** [comment]
Add a Sass file in https://github.com/pradyunsg/furo/tree/main/src/furo/assets/styles/extensions and import it from the index file in there.

In a clean virtualenv, you should be able to do a `pip install -e . "sphinx-theme-builder[cli]", and then run `stb serve <path-to-some-other-docs>` (you might want to install any documentation specific dependencies before ethe stb serve).

I think shipping this with Furo is a better approach, since it avoids any back-and-forth for the responsibility for this.

**Comment by humitos** [comment]
@pradyunsg thanks a lot for such detailed steps!

I followed all of them and work on the fix. Then I realized that the problem is at that project in particular. It seems they are adding some extra CSS that changes the `h1` somehow because. Their `h1` is thinner than the one that comes with `furo` by default --so, I guess they have something else that breaks the tooltip style as well.

This is an example with plain furo setup and no extra CSS added in my own project, which looks great:

![Screenshot_2022-06-22_16-42-00](https://user-images.githubusercontent.com/244656/175058377-7cc74004-dd04-4e2f-881f-e95b3b8fcb13.png)
